ProcessManager.py

```diff
@@ -67,9 +75,9 @@ def stopAllProcesses(self):
 
         print ("Shutting down")
 
-        for process in process_list:
-            process.kill()
-            process.wait()
+        for process in process_dict:
+            process.execute.kill()
+            process.execute.wait()
```

Comment:
`process.execute` is a function (in python, functions are also objects that can be passed around and accessed like any other variable), so this won't work as it is. If you change the `RoboProcesses.execute()` method to store the Popen object as I explained above, you can just replace `execute` with `popen` (or what ever you choose to call it) and it should work the way you'd expect.